go/vt/topo/test/vschema.go

```diff
@@ -48,69 +45,21 @@ func checkVSchema(t *testing.T, ts topo.Impl) {
 	}
 
 	err = ts.SaveVSchema(ctx, "test_keyspace", &vschemapb.Keyspace{
-		Sharded: true,
-		Vindexes: map[string]*vschemapb.Vindex{
-			"stfu1": {
-				Type: "stfu",
-				Params: map[string]string{
-					"stfu1": "1",
-				},
-				Owner: "t1",
-			},
-			"stln1": {
-				Type:  "stln",
-				Owner: "t1",
-			},
-		},
 		Tables: map[string]*vschemapb.Table{
-			"t1": {
-				ColumnVindexes: []*vschemapb.ColumnVindex{
-					{
-						Column: "c1",
-						Name:   "stfu1",
-					}, {
-						Column: "c2",
-						Name:   "stln1",
-					},
-				},
-			},
```

Comment:
Why did you remove the vschema definition here?

Reply:
The new higher level API checks it is correct, and this one wasn't.